builder/builder_utilities.livecodescript

```diff
@@ -555,42 +589,54 @@ private function builderMergExtDownloadedFilePath pMergExtVersion, pEdition
    return builderRepoFolder() & slash & "MergExt-" & pEdition & "-" & pMergExtVersion & ".zip"
 end builderMergExtDownloadedFilePath
 
-command builderMergExtEnsureZip pMergExtVersion, pEdition
-   local tZipPath
-   put builderMergExtDownloadedFilePath(pMergExtVersion, pEdition) into tZipPath
-   if there is not a file tZipPath then
-      local tUrl
-      put builderMergExtUrl(pMergExtVersion, pEdition) into tUrl
-      builderLog "message", "Downloading MergExt to" && tZipPath
-      get shell(merge("curl -o '[[tZipPath]]' '[[tUrl]]'"))
-      if the result is not zero then
-         builderLog "error", "Failed to download MergExt zip:" && it
-         throw "failure"
-      end if
-   end if
-end builderMergExtEnsureZip
+private command builderMergExtUnpack pMergExtVersion, pEdition
+   local tFolder
+   put builderUnpackFolder() & slash & "Ext" into tFolder
+   
+   local tZipFile
+   put builderMergExtDownloadedFilePath(pMergExtVersion, pEdition) into tZipFile
+   
+   local tURL
+   put builderMergExtUrl(pMergExtVersion, pEdition) into tURL
+   
+   builderEnsureZip tZipFile, tURL
+   
+   builderUnpackZip tZipFile, tFolder
+end builderMergExtUnpack
 
-command builderMergExtUnpack pMergExtVersion, pEdition
-   builderMergExtEnsureZip pMergExtVersion, pEdition
+////////////////////////////////////////////////////////////////////////////////
+
+function builderTSNetUrl pTSNetVersion
+   return "https://downloads.livecode.com/tsNet/tsNet_" & pTSNetVersion & ".zip"
```

Comment:
Yes, those need to be cleaned up, I will re-push when I finish some other changes.  I think it would be best to keep to the same format that is being used for mergExt.